src/components/facets/types/Search.vue

```diff
@@ -59,6 +62,17 @@
                         default:
                             this.$root.dispatchToStore('applySearchWithin', value);
                     }
+                }else if(searchBoxConfig){
```

Comment:
The searchbox configuration should be applied only with search mode (ln 57). Also, I think we should keep the current dispatch configuration (without the tab callback and etc.)